### ISSUE TITLE
Auto update exported shows

### ISSUE THREAD

**Issue by snoopyjoe** [Sandmann79/xbmc#131]
I just realized that the only way to update episodes in a show that has been exported is to re-export the show again.

I thought about making a function that would export shows in the library whenever the Watchlist folder is opened. I did something similar in the Psvue add-on. A better idea would be to have a timer run def Export() on the shows saved to the library folder.

I'll keep playing with the code and post any solutions I come up with.

**Comment by Sandmann79** [comment]
For this functionality you can use the Osmosis add-on. With this you can add folders from another add-ons to Kodi's library and periodically update them.

Github: https://github.com/stereodruid/plugin.video.osmosis/tree/future
Forum (German): https://www.kodinerds.net/index.php/Thread/53307-Beta-OSMOSIS-Streams-zur-DB-hinzuf%C3%BCgen/

**Comment by snoopyjoe** [comment]
I tried using OSMOSIS but I kept getting an error. I think maybe because I was trying to export paid content. I didn't look to hard into the error. So instead I just created a cron job to export the watchlist folder. It should run every Monday at 1215a. I thought about adding a xmbc.Monitor to the code but that would just have the addon running in the background forever. Kodi v18 is still alpha so the less addons running in the background the better.

15 0 * * 1 kodi-send --action="RunPlugin(plugin://plugin.video.amazon-test/?mode=getListMenu&url=watchlist&export =1)"
